WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PostRequests/RegisterInputSuccessTests.cs

```diff
@@ -52,8 +52,8 @@ public async Task SuccessFromPreviousCoinJoinAsync()
 		using Key key = new();
 		var coin = WabiSabiFactory.CreateCoin(key);
 		var rpc = WabiSabiFactory.CreatePreconfiguredRpcClient(coin);
-		var coinJoinIdsStore = new InMemoryCoinJoinIdStore();
-		coinJoinIdsStore.Add(coin.Outpoint.Hash);
+		var coinJoinIdsStore = new CoinJoinIdStore();
```

Comment:
Rename `coinJoinIdsStore` to `coinJoinIdStore`?

Reply:
IMO, I expect an `XStore` to contain more than one `X`.
It it was not a `Store`, it could be `Xs`. But thats just my thinking.

Reply:
What's up with this?

Reply:
I totally misunderstood this, fixing.